youtube_dl/utils.py

```diff
@@ -91,6 +91,10 @@ def register_socks_protocols():
     'January', 'February', 'March', 'April', 'May', 'June',
     'July', 'August', 'September', 'October', 'November', 'December']
 
+FRENCH_MONTH_NAMES = [
```

Comment:
@yan12125 OK. But wouldn't it be better to change them to tuples, since they are constant anyway?